package.json

```
@@ -59,5 +59,8 @@
   "engines": {
     "node": ">=18.18.0"
   },
-  "packageManager": "yarn@1.22.19"
+  "packageManager": "yarn@1.22.19",
+  "volta": {
+    "node": "20.17.0"
+  }
```

Let's not include this in this pull request. 
```suggestion
  "packageManager": "yarn@1.22.19"
```

---

Oops thanks for tidying this for me. I need to figure out how to set node version for just this folder without modifying the package json.